HdmiCec_2/HdmiCec_2.cpp

```diff
@@ -141,11 +143,9 @@ namespace WPEFramework
        void HdmiCec_2Processor::process (const Standby &msg, const Header &header)
        {
              printHeader(header);
-             if(header.from.toInt() == LogicalAddress::TV)
-             {
-                 tvPowerState = 1; 
```

Comment:
Why did you remove this code?

Reply:
Hi @apatel859 : 
two reasons
1. tvPowerState variable is not used anywhere else in code in HdmiCec_2. In processing ReportPowerStatus, tvPowerState updated with value, but not used.
2. I see it not correct to update the tvPowerState as 1 when all devices switching to standby.  Please correct me if I am wrong

And please review the pull request:  https://github.com/rdkcentral/rdkservices/pull/1512. 
I have cancelled this and create a review with 2105_sprint